public/app/features/dashboard-scene/edit-pane/DashboardEditPane.tsx

```diff
@@ -2,27 +2,76 @@ import { css } from '@emotion/css';
 import { useEffect, useRef } from 'react';
 
 import { GrafanaTheme2 } from '@grafana/data';
-import { SceneObjectState, SceneObjectBase, SceneObject, SceneObjectRef } from '@grafana/scenes';
-import { ToolbarButton, useStyles2 } from '@grafana/ui';
+import {
+  SceneObjectState,
+  SceneObjectBase,
+  SceneObject,
+  SceneObjectRef,
+  sceneGraph,
+  useSceneObjectState,
+} from '@grafana/scenes';
+import { ElementSelectionContextItem, ElementSelectionContextState, ToolbarButton, useStyles2 } from '@grafana/ui';
 
-import { EditableDashboardElement, isEditableDashboardElement } from '../scene/types';
 import { getDashboardSceneFor } from '../utils/utils';
 
 import { ElementEditPane } from './ElementEditPane';
+import { useEditableElement } from './useEditableElement';
 
 export interface DashboardEditPaneState extends SceneObjectState {
   selectedObject?: SceneObjectRef<SceneObject>;
+  selectionContext: ElementSelectionContextState;
 }
 
 export class DashboardEditPane extends SceneObjectBase<DashboardEditPaneState> {
-  public selectObject(obj: SceneObject) {
+  public constructor() {
+    super({
+      selectionContext: {
+        enabled: false,
+        selected: [],
+        onSelect: (item, multi) => this.selectElement(item, multi),
+      },
+    });
+  }
+
+  public enableSelection() {
+    // Enable element selection
+    this.setState({ selectionContext: { ...this.state.selectionContext, enabled: true } });
+  }
+
+  public disableSelection() {}
```

Comment:
```suggestion
  public disableSelection() {
    this.setState({ selectionContext: { ...this.state.selectionContext, enabled: false } });
  }
```
Why is this not implemented? It is used when unmount but it is missing implementation :)